crates/nu-command/src/debug/metadata_set.rs

```diff
@@ -85,6 +86,11 @@ impl Command for MetadataSet {
                 example: "'crates' | metadata set --datasource-filepath $'(pwd)/crates' | metadata",
                 result: None,
             },
+            Example {
+                description: "Set the metadata of a file path",
+                example: "'crates' | metadata set --content-type text/plain | metadata",
+                result: None,
```

Comment:
Seems like this one could have a testable result because doesn't it just return a record of `content-type text/plain`?

Reply:
Good call, I'll add one.